src/main/webapp/user-page.html

```diff
@@ -30,14 +30,20 @@
         <li><a href="/">Home</a></li>
       </ul>
     </nav>
-    <h1 id="page-title">User Page</h1>
+    <center>
```

Comment:
You shouldn't use center tags. Group everything you want centered in a div and then use CSS to do the centering.

Reply:
The key thing to remember is that tags should be used for structure and CSS should be used for styling.